### ISSUE TITLE
How to copy or render a displayObjectContainer in a map

### ISSUE THREAD

**Issue by michahell** [pixijs/pixi.js#335]
Hi,

I am trying to see what i can do with WebGL by trying to build some sort of top-down landscape generator (but without tilemaps...). i've built a map, which shows you where you are. only, it does not give a reflection of the real world, it's just a Sprite with a texture as background. This is because i do not fully understand how to use RenderTexture yet.

I have a main displayObjectContainer in which i have:
- lots of trees (_Sprites_)
- a transparent PNG texture layer which is interactive (_TilingSprite_)
- a ground surface (_TilingSprite_)

i'd like to reflect all changes in this displayObjectContainer in the map.

Here is a link to the working version #4 : http://www.powergeek.nl/proj/mongento/versions/4/
(Yes, zooming control is still quite buggy, i know that :+1: )

**TL;DR;**
How do i create a 'map' of the real world? that reflects the complete 'display list' ?

I've considered deepcopying the displayObjectContainer in the animate loop, but that seems rather weird and ugly to do? or is that the only option?

Anyway, Pixi is lightning fast and super easy to work with, IMO. keep up the good work, can't wait until there will be shader / filter support!

**Comment by saurfang** [comment]
This is useful. Just to add that it seems now `PIXI.RenderTexture` contstructor requires a renderer  as discribed in https://github.com/pixijs/pixi.js/issues/1893. 

Also I was dumb that I added minimap as a child of the main display object which minimap is based upon causing webgl error because the source and destination overlaps.
